src/aqua_diagnostics/core/cli_base.py

```diff
@@ -147,7 +147,10 @@ def dataset_args(self, dataset):
                 'exp': dataset['exp'], 'source': dataset['source'],
                 'regrid': dataset.get('regrid', self.regrid),
                 'startdate': dataset.get('startdate', None),
-                'enddate': dataset.get('enddate', None)}
+                'enddate': dataset.get('enddate', None),
+                'std_startdate': dataset.get('std_startdate', None),
```

Comment:
If you add this, then ALL diagnostics need to support std_startdate and std_enddate as arguments!